includes/class-wc-payments.php

```diff
@@ -998,7 +998,7 @@ public static function ajax_init_platform_checkout() {
 			'session_cookie_value' => wp_unslash( $_COOKIE[ $session_cookie_name ] ?? '' ), // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
 			'store_data'           => [
 				'store_name'        => get_bloginfo( 'name' ),
-				'store_logo'        => wp_get_attachment_image_src( get_theme_mod( 'custom_logo' ), 'full' )[0] ?? '',
+				'store_logo'        => add_query_arg( 'as_account', '0', get_rest_url( null, 'wc/v3/payments/file/' . self::get_gateway()->get_option( 'platform_checkout_store_logo' ) ) ),
```

Comment:
Now we send full merchant url, this way we can ensure that Plain permalinks work.